README.md

```diff
@@ -1,2 +1,168 @@
+
+### /medications/$medicationId$
+
+Based on [FHIR Medication](https://hl7.org/fhir/R4B/medication.html):
+
+|Property|Type|Values|Comments|
+|-|-|-|-|
+|id|string|-|[Resource](https://hl7.org/fhir/R4B/resource.html): Logical id of this artifact|
+|meta|Meta|-|[Resource](https://hl7.org/fhir/R4B/resource.html#Meta): Metadata about the resource|
+|identifier|list of Identifier|-|Business identifier for this medication|
+|extension|list of Extension|-|See /medications/$medicationId$/extension for possible values|
+
+Technically, we could use [FHIR MedicationKnowledge](https://hl7.org/fhir/R4B/medicationknowledge.html), but it doesn't seem to be ready for use - or at least take over the `medicineClassification` property from it.
+
+### /medications/$medicationId$/extension
+
+Based on the [Extension](https://hl7.org/fhir/R4B/extensibility.html#Extension) format specified in FHIR, a medication may contain a list of these following properties. Each property will need to get a url assigned to fit the FHIR data format.
+
+|Property|Type|Values|Comments|
+|-|-|-|-|
+|medicationClass|string|e.g. "BETABLOCKER"|A medicationClassId referring to a medicationClass specified in /medicationClasses/$medicationClassId$. One medication object may contain multiple medicationClass extension properties.|
+|minimumDailyDose|double|e.g. 6.25|Unit: mg/day. May only occur once.|
+|targetDailyDose|double|e.g. 50.0|Unit: mg/day. May only occur once.|
+
+Q: Do we possibly need a connection from a single medication to a video?
+
+### /medicationClasses/$medicationClassId$
+
+|Property|Type|Values|Comments|
+|-|-|-|-|
+|id|string|e.g. "BETABLOCKER"||
+|name|LocalizedText|-|A name for a given medicationClass to be displayed to a user.|
+|videoSectionId|string|-|To be used to retrieve a video using /videos/$videoSectionId$/videos/$videoId$.|
+|videoId|string|-|To be used to retrieve a video using /videos/$videoSectionId$/videos/$videoId$.|
+
+Q: Do we necessarily need a connection between medication classes and videos / videoSections?
+
+## /videoSections
+
+In this section, we describe all data related to educational videos to be shown in the Engage-HF mobile apps. The videos are grouped into different categories to be displayed as sections in the mobile apps.
+
+### /videoSections/$videoSectionId$
+
+|Property|Type|Values|Comments|
+|-|-|-|-|
+|title|LocalizedText|e.g. "ENGAGE-HF Application"|May be localized.|
+|description|LocalizedText|e.g. "Helpful videos on the ENGAGE-HF mobile application."|May be localized. Are there different videos / videoSections for each platform?|
+
+Q: Can we specify a certain order in Firestore or should we add something like an id to order it by?
+Q: All videos are shown to all users, or is there any filtering/reordering/difference for any users with respect to other users?
```

Comment:
Would be done on the mobile app if needed.

Reply:
Added this to a list of questions. Might be relevant in that sense that it would potentially interfere with the `orderIndex` property or require additional properties to provide context for filtering/reordering.